resources/lang/en/http-statuses.php

```diff
@@ -0,0 +1,84 @@
+<?php
+
+declare(strict_types=1);
+
+return [
+    '0'            => 'Unknown Error',
```

Comment:
```suggestion
    '0' => 'Unknown Error',
```
sửa tương tự các line khác.